userprofile/forms.py

```diff
@@ -57,13 +57,24 @@ def __init__(self, *args, **kwargs):
 
                 if s_val['type'] == 'STRING':
                     if 'options' in s_val:
-                        self.fields[s_field] = forms.ChoiceField(
-                            choices=[
-                                (option, option)
-                                for option in s_val.get('options')
-                            ],
-                            **setting_kwargs,
-                        )
+                        if callable(s_val.get('default')):
+                            self.fields[s_field] = forms.ChoiceField(
+                                choices=[
+                                    (str(option), str(option))
+                                    for option in s_val.get('options')(
```

Comment:
Same comment here as in the project form, I can't find a check for whether `options` are callable.

Reply:
I think here is problem in Github UI)) I've changed it to tuples, see the code section